src/app/game/game.component.ts

```diff
@@ -1,23 +1,32 @@
-import { Component, OnInit } from '@angular/core';
+import { Component, HostListener, OnInit } from '@angular/core';
+import { fromEvent } from 'rxjs';
+import { map } from 'rxjs/operators';
 
-import { Answer } from '../models/answer';
-import { AnswerService } from '../services/answer.service';
+import { GamePlayService } from '../services/game-play.service';
 
 @Component({
   selector: 'app-game',
   templateUrl: './game.component.html',
   styleUrls: ['./game.component.scss']
 })
 export class GameComponent implements OnInit {
-  protected answer: Answer = { id: 0, word: '' };
-  constructor(private answerService: AnswerService) { }
+  constructor(private gameplayService: GamePlayService) { }
+  protected currentGuess: string = '';
+  protected acceptedGuesses: string[] = []
 
   ngOnInit(): void {
-    this.getAnswer();
+    this.gameplayService.currentGuess$.subscribe(currentGuess => this.currentGuess = currentGuess);
+    this.gameplayService.acceptedGuesses$.subscribe(acceptedGuesses => this.acceptedGuesses = acceptedGuesses)
```

Comment:
Inside components, whenever we subscribe to an observable that doesn't complete immediately, we also unsubscribe from that observable in an `ngOnDestroy()` function (this answer is a little long but has an example of how to do it https://stackoverflow.com/a/41177163). That said, you can avoid subscribing/unsubscribing by using the async pipe in the html file `<p>Current Guess: {{ gameplayService.currentGuess$ | async }}</p>`